Documentation/ApiOverview/DependencyInjection/Index.rst

```diff
@@ -442,238 +418,425 @@ Configuration
 
 ..  _dependency-injection-in-extensions:
 
-Configure dependency injection in extensions
---------------------------------------------
+:file:`Services.yaml` declaring service defaults
+------------------------------------------------
 
-Extensions have to configure their classes to make use of the
+Extensions have to configure their classes to make use of
 dependency injection. This can be done in :file:`Configuration/Services.yaml`.
```

Comment:
```suggestion
dependency injection. This can be done in :file:`Configuration/Services.yaml`.
Alternatively, :file:`Configuration/Services.php` can also be used.
```

IMHO we should keep a reference here to allow people understanding that YAML is not the sole way (and quite a few extensions make use of Services.php). I don't think we need a full section about this (yet?) but we shouldn't remove it here....?